src/test/ui/rfcs/rfc-2396-target_feature-11/safe-calls.rs

```diff
@@ -0,0 +1,80 @@
+#![feature(target_feature_11)]
```

Comment:
I would like to see a separate run-pass test for the parts that are not supposed to error.